tests/unit/test_snapshot.py

```diff
@@ -327,12 +327,10 @@ def test_iteration_w_raw_w_multiuse(self):
         resumable = self._call_fut(derived, restart, request)
         self.assertEqual(list(resumable), list(FIRST))
         self.assertEqual(len(restart.mock_calls), 1)
-        begin = 0
-        for call in restart.mock_calls:
-            if "begin" in call.__str__():
-                begin += 1
-
-        self.assertEqual(begin, 1)
+        begin_count = sum(
+            [1 for args in restart.call_args_list if "begin" in args.kwargs.__str__()]
```

Comment:
nit: prefer `str(args.kwargs)` to `args.kwargs.__str__()`. They should be equivalent but the former is more readable.